readme.txt

```diff
@@ -162,6 +162,12 @@ The admin can use the merge code `{{cart.unsubscribe}}' in the email templates.
 
 == Changelog ==
 
+= 6.0.0 ( 10.10.2024 ) =
+* Fix - The test email was not display correctly on iPhone 12 Pro devices.
```

Comment:
@nidhi-tatosaniya **Fix - The test email was not displayed correctly on iPhone 12 Pro devices.**